build.sh

```diff
@@ -3,6 +3,6 @@ docker build -t quay.io/tarilabs/rust:1.31 ./rust/slim
 docker build -t quay.io/tarilabs/rust:alpine-1.40 ./rust/alpine/
 docker build -t quay.io/tarilabs/run:alpine-1.40 ./run/alpine/
 # The nightly version
-docker build -t quay.io/tarilabs/rust_tari-build-with-deps:nightly-2020-09-15 ./rust/build/
+docker build -t quay.io/tarilabs/rust_tari-build-with-deps:nightly-2020-06-10 ./rust/build/
```

Comment:
maybe use `nightly-2020-08-13` to match with everything else? Or was there a build issue?

Reply:
Thanks, copy paste error.